src/components/ActionButton/ActionButton.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../sass/Variables.scss';
+
+$primary-color-lightest: #ffffff;
+$primary-color-lightest-hover: whitesmoke;
+$primary-color-light: #e5e5e5;
+$primary-color-dark: #4a4a4a;
+$accent-color: $calypso;
+
+button.CoveoActionButton.coveo-actionbutton {
+    height: 36px;
```

Comment:
Can we have a variable for this magic 36px ? 😄

Reply:
Yeah sure